kedro/framework/cli/starters.py

```diff
@@ -38,6 +43,81 @@
 }
 _STARTERS_REPO = "git+https://github.com/kedro-org/kedro-starters.git"
 
+
+@define(order=True)
```

Comment:
What does this do?

Reply:
@MerelTheisenQB This is the goodies that `dataclasses` or `attrs` offer. 

Imagine if you have a class like this
```python
class MyClass:
   a: int
   b: int

a = MyClass(1,1)
b = MyClass(1,1)
```
If you do ` a == b`, you get a `False` because it is looking at its hash by default, but what you really want is something like a `namedtuple`, which should look at the values of the field. (We did this for `Node`). But with `dataclasses` and `attrs` class, it will return `True` as it generates the correct `__magic__` method under the hood.

In this case, `order=True` will allows you to do comparison like  `MyClass(1,1) < MyClass(2,1)` by look at the fields value, by default the orders go with `a` then `b`, which makes the class `sortable`. I think under the hood it generates a `__lt__` (lesser than) method.